<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>243</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>243</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>276</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>512</width>
    <height>243</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preta's server properties manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="MainWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0" colspan="6">
     <widget class="QTabWidget" name="tabs">
      <property name="baseSize">
       <size>
        <width>512</width>
        <height>256</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="defaultSetup">
       <attribute name="title">
        <string>Default</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="Ldifficulty">
          <property name="text">
           <string>Difficulty</string>
          </property>
          <property name="buddy">
           <cstring>difficulty</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QSlider" name="difficulty">
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="3">
         <widget class="QLabel" name="LmaxPlayers">
          <property name="text">
           <string>Max Players</string>
          </property>
          <property name="buddy">
           <cstring>maxPlayers</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QSpinBox" name="maxPlayers">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Lgamemode">
          <property name="text">
           <string>Gamemode</string>
          </property>
          <property name="buddy">
           <cstring>gamemode</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QSlider" name="gamemode">
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QCheckBox" name="hardcore">
          <property name="text">
           <string>Hardcore</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="enableCommandBlock">
          <property name="text">
           <string>Command Blocks</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="whitelist">
          <property name="text">
           <string>Whitelist</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QCheckBox" name="enforceWhitelist">
          <property name="text">
           <string>Enforce Whitelist</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QCheckBox" name="onlineMode">
          <property name="text">
           <string>Online</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="LplayerIdleTimeout">
          <property name="text">
           <string>Timeout</string>
          </property>
          <property name="buddy">
           <cstring>playerIdleTimeout</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="playerIdleTimeout">
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QLabel" name="LmaxTickTime">
          <property name="text">
           <string>Tick limit</string>
          </property>
          <property name="buddy">
           <cstring>maxTickTime</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="5" colspan="2">
         <widget class="QSpinBox" name="maxTickTime">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>60000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="Lmotd">
          <property name="text">
           <string>MOTD</string>
          </property>
          <property name="buddy">
           <cstring>motd</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="forceGamemode">
          <property name="text">
           <string>Forced</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="6">
         <widget class="QPlainTextEdit" name="motd">
          <property name="plainText">
           <string>A Minecraft Server</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="permissions">
       <attribute name="title">
        <string>Permissions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="allowFlight">
          <property name="text">
           <string>Allow Flight</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="pvp">
          <property name="text">
           <string>Allow PVP</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="LmaxWorldSize">
          <property name="text">
           <string>Max Size</string>
          </property>
          <property name="buddy">
           <cstring>maxWorldSize</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QSpinBox" name="maxWorldSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>29999984</number>
          </property>
          <property name="value">
           <number>29999984</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="LopPermissionLevel">
          <property name="text">
           <string>OP Level</string>
          </property>
          <property name="buddy">
           <cstring>opPermissionLevel</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="allowNether">
          <property name="text">
           <string>Allow Nether</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QSlider" name="opPermissionLevel">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="spacePerm">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="world">
       <attribute name="title">
        <string>World</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="LlevelName">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="buddy">
           <cstring>levelName</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="levelName">
          <property name="text">
           <string>world</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="LlevelType">
          <property name="text">
           <string>Type</string>
          </property>
          <property name="buddy">
           <cstring>levelType</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QComboBox" name="levelType">
          <item>
           <property name="text">
            <string>Default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Flat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Large Biomes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amplified</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Buffet</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="LlevelSeed">
          <property name="text">
           <string>Seed</string>
          </property>
          <property name="buddy">
           <cstring>levelSeed</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLineEdit" name="levelSeed"/>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="LmaxBuildHeight">
          <property name="text">
           <string>Max height</string>
          </property>
          <property name="buddy">
           <cstring>maxBuildHeight</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QSpinBox" name="maxBuildHeight">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="value">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="LgeneratorSettings">
          <property name="text">
           <string>Generator Settings</string>
          </property>
          <property name="buddy">
           <cstring>generatorSettings</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QLineEdit" name="generatorSettings"/>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="LspawnProtection">
          <property name="text">
           <string>Protection</string>
          </property>
          <property name="buddy">
           <cstring>spawnProtection</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QSpinBox" name="spawnProtection">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="generateStructures">
          <property name="text">
           <string>Structures</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="spawnAnimals">
          <property name="text">
           <string>Animals</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QCheckBox" name="spawnMonsters">
          <property name="text">
           <string>Monsters</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QCheckBox" name="spawnNpcs">
          <property name="text">
           <string>NPC</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="spaceWorld">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="network">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="LserverIp">
          <property name="text">
           <string>IP</string>
          </property>
          <property name="buddy">
           <cstring>serverIp</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="serverIp"/>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="serverPort">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
          <property name="value">
           <number>25565</number>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="LviewDistance">
          <property name="text">
           <string>View distance</string>
          </property>
          <property name="buddy">
           <cstring>viewDistance</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="LserverPort">
          <property name="text">
           <string>Port</string>
          </property>
          <property name="buddy">
           <cstring>serverPort</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="LnetworkCompressionThreshold">
          <property name="text">
           <string>Compression</string>
          </property>
          <property name="buddy">
           <cstring>networkCompressionThreshold</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QSpinBox" name="networkCompressionThreshold">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="1" column="6" colspan="2">
         <widget class="QCheckBox" name="nativeTransport">
          <property name="text">
           <string>Native Transport</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QSpinBox" name="viewDistance">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QCheckBox" name="preventProxyConnections">
          <property name="text">
           <string>Allow Proxy</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="LqueryPort">
          <property name="text">
           <string>Port</string>
          </property>
          <property name="buddy">
           <cstring>queryPort</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="enableRcon">
          <property name="text">
           <string>Enable RCON</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="enableQuery">
          <property name="text">
           <string>Enable Query</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QSpinBox" name="queryPort">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
          <property name="value">
           <number>25565</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="LrconPort">
          <property name="text">
           <string>Port</string>
          </property>
          <property name="buddy">
           <cstring>rconPort</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="2">
         <widget class="QLabel" name="LrconPassword">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>rconPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="5" colspan="2">
         <widget class="QLineEdit" name="rconPassword"/>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QSpinBox" name="rconPort">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
          <property name="value">
           <number>25575</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="spaceNetwork">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="misc">
       <attribute name="title">
        <string>Misc</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QLabel" name="LresourcePackSha1">
          <property name="text">
           <string>sha1</string>
          </property>
          <property name="buddy">
           <cstring>resourcePackSha1</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="resourcePack"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="LresourcePack">
          <property name="text">
           <string>Resource Pack</string>
          </property>
          <property name="buddy">
           <cstring>resourcePack</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="resourcePackSha1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="snooperEnabled">
          <property name="text">
           <string>Snooper</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="spaceMisc">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="credits">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="nameV">
          <property name="text">
           <string>PSPM 1.1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="coded">
          <property name="text">
           <string>Coded by Preta(Shiro Ayame)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="qt">
          <property name="text">
           <string>Used Qt for GUI</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="copyright">
          <property name="text">
           <string>Copyright (C) 2017-2019 CrowZ
Can be used under GPL 3.0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="gitBtn">
          <property name="text">
           <string>Github</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="resetBtn">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="loadBtn">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="spaceRL">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="closeBtn">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="saveBtn">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="spaceSC">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>allowFlight</tabstop>
  <tabstop>allowNether</tabstop>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
